pillar/cove_oc4ids.sls

```diff
@@ -10,9 +18,9 @@ python_apps:
       url: https://github.com/open-contracting/cove-oc4ids.git
     django:
       env:
-        ALLOWED_HOSTS: .standard.open-contracting.org,.oc4ids.opencontracting.uk0.bigv.io
+        ALLOWED_HOSTS: .infrastructure.open-contracting.org,.cove-oc4ids.open-contracting.org
```

Comment:
@jpmckinney, I remember us discussing removing the proxy from the cove servers - which I think will be a good upgrade, simplifying hosting and making server upgrades easier.

What hostnames would you like to use for the cove applications?

Reply:
Proceeding with `review-oc4ids.standard.open-contracting.org` as discussed.